server/channels/app/plugin.go

```diff
@@ -561,20 +561,12 @@ func (a *App) GetMarketplacePlugins(filter *model.MarketplacePluginFilter) ([]*m
 	}
 
 	if !filter.RemoteOnly {
-		// Some plugin don't work on cloud. The remote Marketplace is aware of this fact,
-		// but prepackaged plugins are not. Hence, on a cloud installation prepackaged plugins
-		// shouldn't be shown in the Marketplace modal.
-		// This is a short term fix. The long term solution is to have a separate set of
-		// prepacked plugins for cloud: https://mattermost.atlassian.net/browse/MM-31331.
-		license := a.Srv().License()
-		if license == nil || !license.IsCloud() {
-			appErr := a.mergePrepackagedPlugins(plugins)
-			if appErr != nil {
-				return nil, appErr
-			}
+		appErr := a.mergePrepackagedPlugins(plugins)
+		if appErr != nil {
+			return nil, appErr
 		}
```

Comment:
Cloud gets prepackaged plugins

Reply:
Are we fine merging this change while the list of prepackaged plugins still contains ones that are not supported on Cloud?

Reply:
(re: https://github.com/mattermost/mattermost/pull/24336#discussion_r1305481335) @hanzei was antivirus 1.0.0 already shipped/included in the last few cloud releases due to `hosting: “”`?

Is if necessary, would it be viable to create something like `var cloudExcludedPrepackagedPlugins = []string{	"antivirus"}` akin to: https://github.com/mattermost/mattermost/blob/0f03f2322d7a47bd1615b879ca11d7cdf5151b38/server/channels/app/plugin.go#L1040-L1053

Reply:
Shipping `antivirus` to Cloud via the Marketplace was not intended. Such an exclude list would be fine IMO :+1:

Reply:
Shipping `antivirus` to Cloud via the Marketplace was not intended. But due to https://github.com/mattermost/mattermost/pull/24225/, transitionally prepackaged plugins can't be installed via the Marketplace anyway. So I think we are fine here.